app.js

```diff
@@ -380,3 +382,82 @@ const filterByTags = new FilterByTags();
 class Booking1 {}
 
 class Booking2 {}
+
+class FilterByRating {
+  constructor() {
+    this.ratingRadiosRow1 = document.querySelectorAll("[name='rating_row_1']");
+
+    this.ratingRadiosRow2 = document.querySelectorAll("[name='rating_row_2']");
+
+    this.ratingRadiosRow1.forEach((radio) => {
+      radio.addEventListener("change", () => {
```

Comment:
My only concern with this code is how much work it's doing in the `constructor` of the class. And not only the amount of work, but also the type of work. It's reaching out of the class (to the global `document` object) and makes changes by adding event listeners etc.

The constructor is intended to _construct_ the class instance. I can totally see how someone would argue that adding these event listeners is related to constructing the instance, but I would disagree. I think it would have been nicer to put basically everything that is currently in the constructor in a separate class method called `initialize` or `hookUpToDom` or `listenForEvents` or something like that. That way, using the class would look like this:

```js
const filter = new FilterByRating();
filter.listenForEvents();
```